### ISSUE TITLE
Description as to why each string is "naughty"

### ISSUE THREAD

**Issue by DonaldTsang** [minimaxir/big-list-of-naughty-strings#203]
Would like to read up on how strings can break a system since this is an interesting topic.

**Comment by alimertcakar** [comment]
Some look like they were put there for fun. 
Like: " If you are reading this, you've been in coma for 20 years. We are trying this new technique, you will see this message somewhere randomly. We miss you please wake up."
Ok :P

**Comment by DonaldTsang** [comment]
Yeah it is kinda weird, as some of those dirty techniques should be documented.
